client/components/select-dropdown/style.scss

```diff
@@ -54,11 +54,30 @@
 			right: 13px;
 			top: 12px;
 
+		.is-compact & {
+			right: 4px;
+			top: 4px;
+		}
+
 		display: block;
 		line-height: 18px;
 		color: rgba( $gray, 0.5 );
 	}
 
+
+	.is-compact & {
+		padding: 7px;
+		color: darken( $gray, 10% );
+		font-size: 11px;
+		line-height: 1;
+		text-transform: uppercase;
+
+		.count {
```

Comment:
hmmm, I see your point... the problem is that without this, when the dropdown contains a count component, the height of the dropdown changes because of the height of `<count>`:

Without `.count { position: absolute }`
![image](https://cloud.githubusercontent.com/assets/1554855/11746117/9f4567a6-a01c-11e5-8a20-66d112544721.png)

With it:
![image](https://cloud.githubusercontent.com/assets/1554855/11746104/8f89960c-a01c-11e5-9518-4574bdb37048.png)

Any idea on how to avoid it without the absolute positioning?